Humbertino_Avilez/Day5/First-spring-bootcamp/src/main/java/com/bolsadeideas/springboot/bootcamp/app/services/JavaAnagramsService.java

```diff
@@ -0,0 +1,6 @@
+package com.bolsadeideas.springboot.bootcamp.app.services;
+
+public interface JavaAnagramsService {
+	
+	public String isAnagram(String word, String anagram);
```

Comment:
In Java when you see a method with prefix "is" means that method return a boolean.

Reply:
so??? nothing to say? did u miss the comment?